build.ps1

```diff
@@ -232,6 +251,22 @@ function Invoke-E2ETestSuite {
     Invoke-Step { E2ETests }
 }
 
+function Invoke-TestExecution {
+    param (
+        [ValidateSet("E2ETests","UnitTests",
+        ErrorMessage="Please specify a valid Test Type name from the list.",
+        IgnoreCase=$true)]
+        # File search filter
+        [string]
+        $Filter
+    )
+    switch ($Filter) {
+        E2ETests { Invoke-Step { E2ETests } }
+        UnitTests { Invoke-Step { UnitTests } }
+        Default {"Unknow Test Type"}
+    }
+}
+
```

Comment:
I was thinking in just one `Invoke-TestExecution`method  with different options; just in case we add more testing types "Unit", "API", "Integration", "E2E", etc.